src/bandersnatch/package.py

```diff
@@ -147,15 +147,15 @@ def sync(self, stop_on_error=False, attempts=3):
 
     def _filter_releases(self):
         """
-        Run the release filtering plugins and remove any packages from the
-        packages_to_sync that match any filters.
+        Run the release filtering plugins and remove any releases from
+        `releases` that match any filters.
         """
         versions = list(self.releases.keys())
         for version in versions:
-            filter = False
+            filter_ = False
```

Comment:
What is the reason to change the name of this variable?

Reply:
just to avoid name clashing with the `filter` built-in